src/auto_schedule/auto_schedule.cc

```diff
@@ -293,9 +294,7 @@ std::vector<Ref<Sketch>> AutoSchedule::evolutionarySearch(size_t outSize) {
         init.emplace_back(measuredSketches_[i]);
     }
 
-    std::vector<Ref<Sketch>> v1 = std::move(init);
-    std::vector<Ref<Sketch>> v2;
-    v2.reserve(v1.size());
```

Comment:
Why is `v2` not reserved?

Reply:
Ok I see it's resized later.

Reply:
It will be directly `resize`d. No appending to it.